fvm/src/syscalls/vm.rs

```diff
@@ -34,7 +38,20 @@ pub fn exit(
         "actor aborted".to_owned()
     } else {
         match context.memory.try_slice(message_off, message_len) {
-            Ok(bytes) => String::from_utf8_lossy(bytes).into_owned(),
+            Ok(bytes) => {
+                if bytes.len() > MAX_MESSAGE_LEN {
+                    let prefix = &bytes[..(MAX_MESSAGE_LEN / 2)];
+                    let suffix = &bytes[bytes.len() - (MAX_MESSAGE_LEN / 2)..];
+                    format!(
+                        "{} ... (skipped {} bytes) ... {}",
+                        String::from_utf8_lossy(prefix),
+                        MAX_MESSAGE_LEN - bytes.len(),
```

Comment:
Shouldn't this be the other way around? 

Say `bytes.len()` is 2345, then it will say it skipped (1024-2345) = -1321 bytes, with a negative sign.